### DISCUSSION TITLE
Community Meeting 2022-04-20

### DISCUSSION THREAD

**Discussion by JamieMagee** [microsoft/component-detection#82] [Announcements]
📆 Wednesday 20th April 2022
🕙 10AM Pacific Time ([Find in your timezone](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20220420T17))
☎️ [Teams meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YmVmMjcxMmEtMmYwYi00MGNiLTg4MDgtODNiMDY5ZjNkYjY3%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%22bb41e436-5c77-44e2-b783-1ad2b9f7aa7f%22%7d)

## Agenda

Comment agenda items below

- Demo for SBOM output (#42)

## Attendees

- @JamieMagee (Microsoft)
- @cobya (Microsoft)
- @8gitbrix (Microsoft)
- @grvillic (Microsoft)
- @annaowens (Microsoft)
- @laurajjiang (Microsoft)
- @jcfiorenzano (Microsoft)
- @RushabhBhansali (Microsoft)

## Recording

https://user-images.githubusercontent.com/1358764/164289904-d9ebe4e5-9558-4a4f-ae6d-7f56d9971197.mp4

**Comment by crosscheck350** [comment]
You can check the latest updates through. Its beneficial site[.](www.freefirelovers.com)

**Comment by JamieMagee** [comment]
- [Issues opened since last meeting](https://github.com/microsoft/component-detection/issues?q=is%3Aissue+created%3A%3E2022-03-23)
- [PRs opened since last meeting](https://github.com/microsoft/component-detection/pulls?q=is%3Apr+created%3A%3E2022-03-23)
